src/location.rs

```diff
@@ -40,7 +40,14 @@ impl Location {
     }
 
     pub fn is_rpg_dir(&self) -> bool {
-        self.path == dirs::home_dir().unwrap().join(".rpg")
+        //Home is checked first because that was the default in a previous version
```

Comment:
instead of duplicating logic here, can you make the datafile function public and use that?